### PR TITLE
refactor: move app_bar and rewrite in `RelativeLayout`

### PR DESCRIPTION
**Pull request by tokitou-san**

### CODE REVIEW

File: src/libs/screens/home_page.py [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
         super().__init__(*args, **kwargs)
 
-        self.add_widget(self.layout())
+        self.md_bg_color = get_color_from_hex("#03020c")
+        self.add_widget(self.app_bar())
 
-    def layout(self):
+    # AppBar layout
+    def app_bar(self) -> MDBoxLayout:
+        Widget()
```
Comment (baseplate-admin) [inline src/libs/screens/home_page.py:18]: Why is there this empty widget?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/libs/screens/home_page.py [hunk 2]
```diff
@@ -1,20 +1,69 @@
...
+                "center_y": 0.5,
+            },
+        )
+        # Add widgets to the main Appbar layout
+        app_bar_layout.add_widget(core_logo)
+        app_bar_layout.add_widget(appbar_right)
+        # Add other widgets
+        appbar_right.add_widget(search_button)
+        appbar_right.add_widget(notification_button)
+        appbar_right.add_widget(profile_button)
```
Comment (baseplate-admin) [inline src/libs/screens/home_page.py:67]: Do we want to make it a  **reusable** functional component?

Comment (tokitou-san) [reply src/libs/screens/home_page.py:67]: > Do we want to make it a reusable functional component?

No? we dont need this appbar everywhere ( just only on homepage )

Comment (baseplate-admin) [reply src/libs/screens/home_page.py:67]: I think its Better to add this to a function

Comment (tokitou-san) [reply src/libs/screens/home_page.py:67]: uhh like in `src/libs/components/` ?

Comment (baseplate-admin) [reply src/libs/screens/home_page.py:67]: yes..

**Review by baseplate-admin — APPROVED** [review]

**Pull request merged**